.github/dependabot.yml

```
@@ -21,4 +21,6 @@ updates:
       # Releases too often, it's annoying
       - dependency-name: "org.assertj:*"
         update-types: ["version-update:semver-patch"]
-
+      # We don't want to stay on an LTS version and want to have more regular updates here:
+      - dependency-name: "io.quarkiverse.openapi.generator:*"
+        versions: ["2.6.0-lts"]
```

On my phone, sorry, but would this work?

```suggestion
        versions: ["*-lts"]
```

---

Let me try it on my fork 😃 ... but if nothing changed on the Dependabot's side:
https://github.com/hibernate/hibernate-search/blob/0604e929f9dd2787adf47ad9c95a9a2fff24a5b8/.github/dependabot.yml#L120-L121
patterns weren't supported 😕

---

yeah...

> The property '#/updates/0/ignore/1/versions' includes invalid version requirements for a maven ignore condition

does not look like patterns work here 😕 😃

---

Thanks for trying :)